### PR TITLE
Simplified the Nox Session Names

### LINKED ISSUE #120: Simplify the Nox session names
**Is your feature request related to a problem? Please describe.**

Both the GitHub CI and users depend on Nox for convenience stuff. We use it regularly for linting, formatting, and building dists. Though the session names aren't that long, it would be great to reduce them a bit more.

**Describe the solution you'd like**

A while back, I proposed the following name change in Discord:

- `check-quality` -> `lint`
- `keep-codebase-clean` -> `format`
- `create-packages` -> `package`

Doing this should be quite simple, it's just about changing the function names in `noxfile.py`. For example, here's the `keep-codebase-clean` name:

https://github.com/DiddiLeija/diddi-and-the-bugs/blob/4e0618b10302943940be937e785f8408011b4918/noxfile.py#L16-L17

**Describe alternatives you've considered**

Keep as is. Again, it's not a big deal, but an improvement would be nice too ;)

**Additional context**

I've tagged this issue as a **good first issue**. I would like a contribution from a new folk, and I think this issue is a great start (even for a one-shot contribution). Of course, if no one takes this, I'll do it.

### PR DESCRIPTION
**Pull request by MarxML**
#### Response to issue opened by @DiddiLeija "Simplify the Nox session names #120"

For ease of use, I've changed 
> check-quality -> lint
> keep-codebase-clean -> format
> create-packages -> package

I also deleted a line break between imports. 

This is a cool looking game! Thanks for opening such a simple issue so I could contribute to it :)

Closes #120.

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -5,33 +5,32 @@
 
 import os
 import sys
-
```
Comment (DiddiLeija) [inline noxfile.py:8]: Actually I'd revert this change, since isort will be unhappy with this :)

File: noxfile.py [hunk 2]
```diff
@@ -5,33 +5,32 @@
...
 
-nox.options.sessions = ["keep-codebase-clean", "check-quality"]
+nox.options.sessions = ["format", "lint"]
 
 files = ["noxfile.py", "main.py", "setup.py"]
 
 
-@nox.session(name="keep-codebase-clean")
-def keep_codebase_clean(session):
+@nox.session(name="format")
```
Comment (DiddiLeija) [inline noxfile.py:15]: It seems to me that these lines are no longer needed, could you please remove all the `@nox.session(...)` lines?

**Review by DiddiLeija — COMMENTED** [review]
Hi! Thanks for contributing! :tada:

I've just added a few comments about linting & a small improvement ;)

**Comment by DiddiLeija** [comment]
BTW, I linked this PR to the related issue, so both tickets will get resolved automatically :)

**Review by MarxML — COMMENTED** [review]
I removed all the necessary lines and the line break on 8 as requested. 

On a side note, I'm new to GitHub, so if this is the incorrect or inconvenient way to update this pr please let me know!

**Comment by DiddiLeija** [comment]
Ok! Thanks!

> On a side note, I'm new to GitHub, so if this is the incorrect or inconvenient way to update this pr please let me know!

Don't worry, I'd say that's the right way by now :)

**Comment by DiddiLeija** [comment]
Hmm... Now I'm investigating the GitHub checks failure:

```
Run nox --non-interactive -s check-quality
nox > No sessions found in /home/runner/work/diddi-and-the-bugs/diddi-and-the-bugs/noxfile.py.
Error: Process completed with exit code 3.
```

Ohh... I forgot that before, you may have to update this line from another file:

https://github.com/MarxML/diddi-and-the-bugs-copy/blob/b6076f26390e6bfbf752aced774b01a67096c985/.github/workflows/ci.yml#L36

We're now invoking an unexistent session (`check-quality`, which is now `lint`), so we must change the command we're running on that line. Could you make this last change before merging?

File: noxfile.py [hunk 3]
```diff
@@ -8,30 +8,27 @@
...
 
-nox.options.sessions = ["keep-codebase-clean", "check-quality"]
+nox.options.sessions = ["format", "lint"]
 
 files = ["noxfile.py", "main.py", "setup.py"]
 
 
-@nox.session(name="keep-codebase-clean")
-def keep_codebase_clean(session):
+def format(session):
```
Comment (DiddiLeija) [inline noxfile.py:16]:
```suggestion
@nox.session
def format(session):
```

File: noxfile.py [hunk 4]
```diff
@@ -8,30 +8,27 @@
...
     "Run formatters."
     session.install("-r", "test-requirements.txt")
     session.run("isort", *files)
     session.run("black", *files)
 
 
-@nox.session(name="check-quality")
-def check_quality(session):
+def lint(session):
```
Comment (DiddiLeija) [inline noxfile.py:23]:
```suggestion
@nox.session
def lint(session):
```

File: noxfile.py [hunk 5]
```diff
@@ -8,30 +8,27 @@
...
     "Check the style and quality."
     session.install("-r", "test-requirements.txt")
     session.run("flake8", *files, "--max-line-length=127")
     session.run("isort", "--check-only", *files)
     session.run("black", "--check", *files)
 
 
-@nox.session(name="create-packages")
-def create_packages(session):
+def package(session):
```
Comment (DiddiLeija) [inline noxfile.py:31]:
```suggestion
@nox.session
def package(session):
```

**Review by DiddiLeija — COMMENTED** [review]
Seems like removing the `nox.session` lines was my mistake. I've added the suggestions in the GitHub format (to commit the changes, just click the "Commit suggestion" buttons), and please sorry for the confusion :/

**Comment by MarxML** [comment]
That misunderstanding was on me, sorry about that. Thanks for clearing it up!

**Review by DiddiLeija — APPROVED** [review]
It's finally done! Thank you for contributing @MarxML! :rocket:

**Comment by DiddiLeija** [comment]
Ok, seems like it's ok to merge this :)
Thanks again!
